### PR TITLE
debugging paths.

### PR DESCRIPTION
**Pull request by detroitpro**
- started logging, 
- reviewed each path
- fixed bug with saving settings
- renamed paths
- added app path
- default size to 800x600
- added footer componet

### CODE REVIEW

File: src/app/components/masthead/masthead.component.ts [hunk 1]
```diff
@@ -7,7 +8,7 @@ import { Component, OnInit } from '@angular/core';
 })
 export class MastheadComponent implements OnInit {
 
-  constructor() { }
+  constructor(private router: Router) { }
```
Comment (detroitpro) [inline src/app/components/masthead/masthead.component.ts]: looks like I didn't finish making the logo nav back to wallets screen.

File: src/app/providers/akroma-client.service.ts [hunk 2]
```diff
@@ -42,25 +45,34 @@ export class AkromaClientService {
     try {
       this.client = clientConstants.clients.akroma.platforms[this.es.os.platform()][this.es.os.arch()];
       settings = await this.settingsService.db.get('system');
+      await this.settingsService.db.remove(settings);
```
Comment (detroitpro) [inline src/app/providers/akroma-client.service.ts]: leaving this in until paths are working as expected. not too many people will be changing their paths.

File: src/app/providers/akroma-client.service.ts [hunk 3]
```diff
@@ -42,25 +45,34 @@ export class AkromaClientService {
...
+    this.syncMode = settings.syncMode;
+    callback();
   }
 
-  async defaultSettings() {
-    console.log('loading default settings');
-    const result = await this.settingsService.db.put({
+  async defaultSettings(): Promise<any> {
+    this.logger.debug('loading default settings');
+    const saveMe = {
```
Comment (detroitpro) [inline src/app/providers/akroma-client.service.ts]: I'd like to do better with using types.

File: main.ts [hunk 4]
```diff
@@ -22,8 +23,8 @@ function createWindow() {
   win = new BrowserWindow({
     x: 0,
     y: 0,
-    width: size.width,
-    height: size.height,
+    width: 800,
+    height: 600,
```
Comment (StephenKoller) [inline main.ts]: seems like a sensible starting point for desktop

File: src/app/components/masthead/masthead.component.ts (hunk 1 above)
Comment (StephenKoller) [reply src/app/components/masthead/masthead.component.ts]: @detroitpro want to push a quick commit in to do just that?

File: src/app/providers/akroma-client.service.ts (hunk 3 above)
Comment (iandesj) [reply src/app/providers/akroma-client.service.ts]: `SystemSettings` is a type, you should check it out. Something like ```const saveMe: SystemSettings = {
...
};``` would be good!

Comment (iandesj) [reply src/app/providers/akroma-client.service.ts]: Just implement that interface that way.

File: src/app/providers/akroma-logger.service.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
+import { Injectable, OnInit } from '@angular/core';
+import { ElectronService } from './electron.service';
+import { SettingsPersistenceService } from './settings-persistence.service';
+
+@Injectable()
+export class AkromaLoggerService implements OnInit {
```
Comment (iandesj) [inline src/app/providers/akroma-logger.service.ts]: :ok_hand:

File: src/styles.scss [hunk 6]
```diff
@@ -10,9 +10,9 @@ html, body {
   -moz-background-size: cover;
   -o-background-size: cover;
 
-  padding-left: 85px;
-  padding-right: 85px;
-  padding-top: 85px;
+  // padding-left: 85px;
+  // padding-right: 85px;
+  // padding-top: 85px;
```
Comment (StephenKoller) [inline src/styles.scss]: If these lines aren't needed, I'd prefer to just remove them outright.

File: src/app/components/masthead/masthead.component.ts (hunk 1 above)
Comment (detroitpro) [reply src/app/components/masthead/masthead.component.ts]: bitmoji

**Pull request merged**